src/main/io/osd.h

```diff
@@ -28,10 +28,15 @@ extern const char * const osdTimerSourceNames[OSD_NUM_TIMER_TYPES];
 
 #define OSD_ELEMENT_BUFFER_LENGTH 32
 
+#define OSD_PROFILE_COUNT 3
+
 #define VISIBLE_FLAG  0x0800
 #define VISIBLE(x)    (x & VISIBLE_FLAG)
 #define OSD_POS_MAX   0x3FF
-#define OSD_POSCFG_MAX   (VISIBLE_FLAG|0x3FF) // For CLI values
+#define OSD_POSCFG_MAX   (OSD_PROFILE_FLAGS|VISIBLE_FLAG|0x3FF) // For CLI values
+#define OSD_PROFILE_FLAGS 0x7000
```

Comment:
`(((1 << OSD_PROFILE_COUNT) - 1) << 12)`

Reply:
Why over complicate this?

Reply:
Answer: To improve maintainability. Hard coded 'magic numbers' are prone to break when things around them are changed, and should be avoided for things that can be evaluated from other values at compile time.

Reply:
Done.